lib/rules/property-no-unknown/README.md

```diff
@@ -127,6 +127,27 @@ The following patterns are _not_ considered violations:
 }
 ```
 
+### `ignoreAtRules: ["/regex/", /regex/, "string"]`
+
+Skips checking properties of the given at-rules.
+
+Given:
+
+```
+["supports"]
+```
+
+The following patterns are _not_ considered violations:
+
+<!-- prettier-ignore -->
+```css
+@supports(display: my-property) {
+  div {
+      display: my-property
```

Comment:
[nipick] Thanks for fixing the indentation, but it remains:

```suggestion
    display: my-property
```

Reply:
Whoops! I'll update my editor's spacing defaults and fix.